specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-11-15-preview/managedCassandra.json

```diff
@@ -1345,6 +1538,81 @@
         }
       }
     },
+    "CommandPublicResource": {
+      "description": "resource representing a command",
+      "type": "object",
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "The command which should be run"
+        },
+        "commandId": {
+          "type": "string",
+          "description": "The unique id of command"
+        },
+        "arguments": {
+          "type": "object",
+          "description": "The arguments for the command to be run",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "host": {
+          "type": "string",
+          "description": "IP address of the cassandra host to run the command on"
+        },
+        "isAdmin": {
+          "type": "boolean",
+          "description": "Whether command has admin privileges"
+        },
+        "cassandraStopStart": {
+          "type": "boolean",
```

Comment:
seems like to control different behaviors, please consider using enum instead:
https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md

Reply:
I believe a boolean would work here, as it is either "true", meaning Cassandra requires it to stop, run command, and start again, or "false", meaning that a stop is not required to run command. There wouldn't need to be any additional actions besides the two.